submissions/src/main/kotlin/gov/cdc/prime/reportstream/submissions/config/AzureConfig.kt

```diff
@@ -0,0 +1,58 @@
+package gov.cdc.prime.reportstream.submissions.config
+
+import com.azure.data.tables.TableClient
+import com.azure.data.tables.TableClientBuilder
+import com.azure.storage.blob.BlobServiceClient
+import com.azure.storage.blob.BlobServiceClientBuilder
+import com.azure.storage.queue.QueueServiceClient
+import com.azure.storage.queue.QueueServiceClientBuilder
+import com.microsoft.applicationinsights.TelemetryClient
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+class AzureConfig {
+
+    @Value("\${azure.storage.connection-string}")
+    private lateinit var connectionString: String
+
+    @Value("\${azure.storage.container-name}")
+    private lateinit var containerName: String
+
+    @Value("\${azure.storage.queue-name}")
+    private lateinit var queueName: String
+
+    @Value("\${azure.storage.table-name}")
+    private lateinit var tableName: String
+
+    @Bean
+    fun blobServiceClient(): BlobServiceClient = BlobServiceClientBuilder()
```

Comment:
Any reason to make this the `bean` rather than the blobcontainer client?

Reply:
moved all of them to the client instead of the service. Definitely cleaner